pkg/controllers/managementuser/settings/controller.go

```diff
@@ -2,59 +2,44 @@ package settings
 
 import (
 	"context"
-	"fmt"
 
-	v3 "github.com/rancher/rancher/pkg/apis/management.cattle.io/v3"
 	"github.com/rancher/rancher/pkg/settings"
+
+	apis "github.com/rancher/rancher/pkg/apis/management.cattle.io/v3"
+	v3 "github.com/rancher/rancher/pkg/generated/norman/management.cattle.io/v3"
 	"github.com/rancher/rancher/pkg/types/config"
-	"github.com/rancher/wrangler/pkg/apply"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
 )
 
-var (
-	toCopy = map[string]bool{
-		settings.InstallUUID.Name:     true,
-		settings.IngressIPDomain.Name: true,
-	}
-)
+var toCopy = map[string]bool{}
 
 type handler struct {
+	cluster   v3.ClusterController
 	clusterID string
-	apply     apply.Apply
 }
 
 func Register(ctx context.Context, userContext *config.UserContext) error {
-	apply, err := apply.NewForConfig(&userContext.RESTConfig)
-	if err != nil {
-		return err
-	}
-
 	h := &handler{
+		cluster:   userContext.Management.Management.Clusters("").Controller(),
 		clusterID: userContext.ClusterName,
-		apply:     apply.WithDynamicLookup().WithCacheTypes(userContext.Management.Wrangler.Mgmt.Setting()),
 	}
 
 	userContext.Management.Management.Settings("").AddHandler(ctx, "copy-settings", h.onChange)
 	return nil
 }
 
-func (h *handler) onChange(key string, obj *v3.Setting) (runtime.Object, error) {
+func (h *handler) onChange(key string, obj *apis.Setting) (runtime.Object, error) {
 	if obj == nil || !toCopy[obj.Name] {
 		return nil, nil
 	}
+	h.cluster.Enqueue("", h.clusterID)
```

Comment:
Handle the error here.

Reply:
`Enqueue` is actually a void method, which I found surprising at first but I'm assuming it has to do with the asynchronous nature of en-queuing objects.

Reply:
Oops. Sorry about that.